### PR TITLE
Updated to Django Oscar 1.3 

### PR DESCRIPTION
**Pull request by clintonb**
ECOM-7095

### CODE REVIEW

**Comment by clintonb** [comment]
@edx/ecommerce please review

File: ecommerce/coupons/tests/mixins.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ class CouponMixin(object):
 
     def setUp(self):
         super(CouponMixin, self).setUp()
-        self.category = factories.CategoryFactory()
+        self.category = factories.CategoryFactory(path='1000')
```
Comment (mikedikan) [inline ecommerce/coupons/tests/mixins.py]: Curious about this addition - do you think it's worthy of a comment?

File: ecommerce/extensions/catalogue/tests/mixins.py [hunk 2]
```diff
@@ -22,13 +23,19 @@ class CourseCatalogTestMixin(object):
...
     useful when running tests without database migrations (which normally create these objects).
     """
+
     def setUp(self):
         super(CourseCatalogTestMixin, self).setUp()
 
         # Force the creation of a seat ProductClass
         self.seat_product_class  # pylint: disable=pointless-statement
         self.enrollment_code_product_class  # pylint: disable=pointless-statement
-        self.category, _created = Category.objects.get_or_create(name='Seats', defaults={'depth': 1})
```
Comment (rlucioni) [inline ecommerce/extensions/catalogue/tests/mixins.py]: What was wrong with the get_or_create?

File: ecommerce/extensions/dashboard/app.py [hunk 3]
```diff
@@ -1,7 +1,13 @@
+from auth_backends.urls import auth_urlpatterns
 from django.conf.urls import url, include
 from oscar.apps.dashboard import app
 from oscar.core.loading import get_class
 
+from ecommerce.core.views import LogoutView
+
+# NOTE: This should match AUTH_URLS in ecommerce/urls.py.
+AUTH_URLS = [url(r'^logout/$', LogoutView.as_view(), name='logout'), ] + auth_urlpatterns
```
Comment (rlucioni) [inline ecommerce/extensions/dashboard/app.py]: Just curious, why does this need to appear here in addition to ecommerce/urls.py? Can you import the variable and use it here?

**Review by rlucioni — APPROVED** [review]

**Comment by rlucioni** [comment]
@clintonb have you taken a look at the migrations being added to see if any will be problematic with our data? I see some which touch the basket and order tables.

**Comment by clintonb** [comment]
@rlucioni the migration changes are mostly Django-related (e.g. modify `verbose_name`). They will pose no problem for out data.

**Pull request merged**